monitoring/multitenant/tls/mt-grafana-tls-values.yaml

```diff
@@ -16,7 +16,7 @@ extraContainers: |
     - --key=/cert/tls.key
     - --cert=/cert/tls.crt
     - --disable-authentication
-    image: squareup/ghostunnel:v1.5.2
+    # image: squareup/ghostunnel:v1.5.2
     imagePullPolicy: IfNotPresent
```

Comment:
I'm not sure why this was commented out.  It might've been a mishap from when I was checking into the "native TLS" ticket. Will remove it in the next push.